databricks/koalas/frame.py

```diff
@@ -2541,7 +2541,7 @@ def astype(self, dtype) -> 'DataFrame':
         return DataFrame(sdf, self._metadata.copy())
 
     # TODO: percentiles, include, and exclude should be implemented.
-    def describe(self) -> 'DataFrame':
+    def describe(self, percentiles=None) -> 'DataFrame':
```

Comment:
Could you add a type hint? `percentiles: Optional[List[float]] = None`

Reply:
Done